resources/prefs.json

```diff
@@ -8,7 +8,7 @@
   "dom.gamepad.enabled": false,
   "dom.microdata.testing.enabled": true,
   "dom.mouseevent.which.enabled": false,
-  "dom.mutation_observer.enabled": false,
```

Comment:
This should be disabled by default, when running tests it can be enabled via:
`./mach run -d -- --pref dom.mutation_observer.enabled`

Reply:
Oh alright 👍